sample-apps/fdb-profile-analyzer/sample_template.yaml

```diff
@@ -0,0 +1,120 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  labels:
+    app: fdb-profile-analyzer
+  name: {{ .JobName }}
+  namespace: {{ .Namespace }}
+spec:
+  backoffLimit: 6
+  completionMode: NonIndexed
+  completions: 1
+  parallelism: 1
+  suspend: false
+  template:
+    metadata:
+      labels:
+        app: fdb-profile-analyzer
+    spec:
+      initContainers:
+        - args:
+            - --copy-library
+            - "6.2"
+            - --copy-binary
+            - fdbcli
+            - --copy-binary
+            - fdbbackup
+            - --copy-binary
+            - fdbrestore
+            - --output-dir
+            - /var/output-files/6.2.30
+            - --init-mode
+          image: foundationdb/foundationdb-kubernetes-sidecar:6.2.30-1
+          name: foundationdb-kubernetes-init-6-2
+          volumeMounts:
+            - mountPath: /var/output-files
+              name: fdb-binaries
+        - args:
+            - --copy-library
+            - "6.3"
+            - --copy-binary
+            - fdbcli
+            - --copy-binary
+            - fdbbackup
+            - --copy-binary
+            - fdbrestore
+            - --output-dir
+            - /var/output-files/6.3.24
+            - --init-mode
+          image: foundationdb/foundationdb-kubernetes-sidecar:6.3.24-1
+          name: foundationdb-kubernetes-init-6-3
+          volumeMounts:
+            - mountPath: /var/output-files
+              name: fdb-binaries
+        - args:
+            - --copy-library
+            - "7.1"
+            - --copy-binary
+            - fdbcli
+            - --copy-binary
+            - fdbbackup
+            - --copy-binary
+            - fdbrestore
+            - --output-dir
+            - /var/output-files/7.1.19
+            - --init-mode
+          image: foundationdb/foundationdb-kubernetes-sidecar:7.1.19-1
+          name: foundationdb-kubernetes-init-7-1
+          volumeMounts:
+            - mountPath: /var/output-files
+              name: fdb-binaries
+      containers:
+        - command:
+            - /bin/bash
```

Comment:
Is there a reason we use bash here and not directly Python3?

Reply:
Will be worked out in another issue.

Reply:
Created issue: https://github.com/FoundationDB/fdb-kubernetes-operator/issues/1348